modules/appeals_api/spec/docs/v2/hlr_spec.rb

```diff
@@ -353,5 +353,166 @@
+
+      response '200', 'Valid' do
+        let(:hlr_body) do
+          JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'valid_200996_minimum_v2.json')))
+        end
+
+        schema JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'hlr_validate.json')))
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
+
+        it 'minimum fields used' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '200', 'Valid' do
+        let(:hlr_body) do
+          JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'valid_200996_v2.json')))
+        end
+
+        schema JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'hlr_validate.json')))
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
+
+        it 'all fields used' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '422', 'Error' do
+        schema JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'errors',
+                                                                 'default.json')))
+        let(:hlr_body) do
+          request_body = JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'valid_200996_v2.json')))
+          request_body['data']['attributes'].delete('informalConference')
+          request_body
+        end
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
+
+        it 'Violates JSON schema' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '422', 'Error' do
```

Comment:
Not necessary now, but I'd like to see these more common errors that are the same across our endpoints made into support helpers so we could do things like `it_behaves_like 'Violates JSON Schema'`, etc :)